### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by Aduda-Boaz**
This PR is for the Leaderboard project setup. The project is built using:
- HTML
- Git Workflows
- JS
- Webpack
The project is structured and working as expected.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> The Project uses APIs to show scores of a game.
+
+![screenshot](./src/leaderboard.png)
+
+Additional description about the project and its features.
```
Comment (mikethreels) [inline README.md]: - [x] please remove the placeholder text in this line and put a short description about the project in its place.

File: README.md [hunk 2]
```diff
@@ -1 +1,67 @@
...
+- HTML5
+- CSS
+- Git Workflows
+- Webpack
+- ES6
+- VS code
+
+## Live Demo
+
+<!-- [Live Demo Link](https://livedemo.com) -->
```
Comment (mikethreels) [inline README.md]: - [x] please remove the entire header in case a part of the template readme is not required.

File: README.md [hunk 3]
```diff
@@ -1 +1,67 @@
...
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
```
Comment (mikethreels) [inline README.md]: - [x] please remove the placeholder text in this line.

File: README.md [hunk 4]
```diff
@@ -1 +1,67 @@
...
+To get a local copy up and running follow these simple example steps.
+
+### Clone
+
+Clone the project from Github <https://github.com/Aduda-Boaz/leaderboard.git> to your local computer
+
```
Comment (mikethreels) [inline README.md]: - [x] please make instructions as if someone who is relatively new to this is going to clone your project and try to install and use it.

File: README.md [hunk 5]
```diff
@@ -1 +1,67 @@
...
+- GitHub: [@Aduda-Boaz](https://github.com/Aduda-Boaz)
+- Twitter: [@BoazAduda](https://twitter.com/BoazAduda)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/boaz-aduda)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
```
Comment (mikethreels) [inline README.md]: -  [x] this link is not working make sure it is referring to the intended file or page, as in [the template](https://github.com/microverseinc/readme-template) provided by microverse

File: README.md [hunk 6]
```diff
@@ -1 +1,67 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](lic.url) licensed.
```
Comment (mikethreels) [inline README.md]: -  [ ] this link is not working make sure it is referring to the intended file or page, as in [the template](https://github.com/microverseinc/readme-template) provided by Microverse

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Boaz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by o-rashad — APPROVED** [review]
Asslam Alaikum @Aduda-Boaz ,😊

Hi👋🏽, Your project is complete!  There is nothing else to say other than... it's time to merge it 😎 

# Congratulations! 🎉 

Happy coding!👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**